src/Types.hs

```diff
@@ -518,7 +520,8 @@ newState rs i u m tz hist sp = ChatState
   , _csChannelSelectState          = emptyChannelSelectState
   , _csRecentChannel               = Nothing
   , _csUrlList                     = list UrlList mempty 2
-  , _csConnectionStatus            = Disconnected
+  , _csConnectionStatus            = Connected
```

Comment:
Why did the initial connection status change to `Connected`? I think we rely on the `Disconnected` -> `Connected` transition at startup.

Reply:
It is set to `Connected` in the develop branch https://github.com/matterhorn-chat/matterhorn/blob/develop/src/Types.hs#L518. I had changed it `Disconnected` in a previous commit but I changed it back in this commit.

Reply:
Oh, I see - thanks!